src/installer/pkg/sfx/Microsoft.NETCore.App/Directory.Build.props

```diff
@@ -174,6 +174,20 @@
     <PlatformManifestFileEntry Include="Microsoft.DiaSymReader.Native.amd64.dll" IsNative="true" FallbackFileVersion="$(MicrosoftDiaSymReaderNativeFileVersion)" />
     <PlatformManifestFileEntry Include="Microsoft.DiaSymReader.Native.arm.dll" IsNative="true" FallbackFileVersion="$(MicrosoftDiaSymReaderNativeFileVersion)" />
     <PlatformManifestFileEntry Include="Microsoft.DiaSymReader.Native.arm64.dll" IsNative="true" FallbackFileVersion="$(MicrosoftDiaSymReaderNativeFileVersion)" />
+    <!-- NativeAOT specific files -->
+    <PlatformManifestFileEntry Include="libbootstrapper.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libbootstrapperdll.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libeventpipe-disabled.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libeventpipe-enabled.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libnumasupportdynamic.a" IsNative="true" />
+    <PlatformManifestFileEntry Include="libnumasupportstatic.a" IsNative="true" />
```

Comment:
```suggestion
```
Not needed since https://github.com/dotnet/runtime/commit/de5a04b2d7d184c5a9121b56cd51c713f72def76.

```sh
$ ls /root/.nuget/packages/runtime.linux-musl-arm64.microsoft.dotnet.ilcompiler/8.0.0-preview.4.23217.4/sdk |\
     grep -vE '\.(xml|pdb)' |\
     while read name; do echo '   <PlatformManifestFileEntry Include="'$name'" IsNative="true" />'; done

   <PlatformManifestFileEntry Include="System.Private.CoreLib.dll" IsNative="true" />
   <PlatformManifestFileEntry Include="System.Private.DisabledReflection.dll" IsNative="true" />
   <PlatformManifestFileEntry Include="System.Private.Reflection.Execution.dll" IsNative="true" />
   <PlatformManifestFileEntry Include="System.Private.StackTraceMetadata.dll" IsNative="true" />
   <PlatformManifestFileEntry Include="System.Private.TypeLoader.dll" IsNative="true" />
   <PlatformManifestFileEntry Include="libRuntime.ServerGC.a" IsNative="true" />
   <PlatformManifestFileEntry Include="libRuntime.WorkstationGC.a" IsNative="true" />
   <PlatformManifestFileEntry Include="libbootstrapper.a" IsNative="true" />
   <PlatformManifestFileEntry Include="libbootstrapperdll.a" IsNative="true" />
   <PlatformManifestFileEntry Include="libeventpipe-disabled.a" IsNative="true" />
   <PlatformManifestFileEntry Include="libeventpipe-enabled.a" IsNative="true" />
   <PlatformManifestFileEntry Include="libstdc++compat.a" IsNative="true" />
```

Reply:
awesome, thanks!